### DISCUSSION TITLE
Multiple UERANSIM UEs

### DISCUSSION THREAD

**Discussion by abouillot-stl** [herlesupreeth/docker_open5gs#170] [Q&A]
Hello,

The docker setup comes with one UE configured in the UERANSIM setup, if I correctly understand.

I'm planning to explore 5GNR slicing and would like to have various config at hand separately or simultaneously.

What would be you recommendation in order to have multiple UEs pools with different configs?

Thanks,
Alexandre

**Comment by herlesupreeth** [comment]
You can connect multiple UEs using the UE count or something like that in UERANSIM. I believe it will increment the IMSI by 1 until the UE count is reached.

When you mean slicing, can you elaborate on the scenario you are trying to achieve?

**Comment by abouillot-stl** [comment]
I noticed the multi UE option in UERANSIM, but they will all share the same configuration.

I would like to explore the coexistence of different UEs with different slicing configuration - i.e. normal usage; personal/professional usage; IoT - simultaneously.

As I understand each of these groups would require a different `open5gs-ue.yaml` file for the various slices.

**Reply by herlesupreeth** [reply]
> I would like to explore the coexistence of different UEs with different slicing configuration - i.e. normal usage; personal/professional usage; IoT - simultaneously.
> As I understand each of these groups would require a different open5gs-ue.yaml file for the various slices.

Thats right. To test this scenario it wont be straightforward, you would have to create separate open5gs-ue.yaml and modify the ueransim_image_init.sh and create nr-ue.yaml (docker-compose) file for each
